thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -116,6 +113,18 @@ export default Ember.Component.extend({
     return this.get('status') === 'NO_FEEDBACK';
   }),
 
+  isHidden: Ember.computed(
+    'isNeedFeedback',
+    'isHiddenUser',
+    function () {
+      const { isNeedFeedback, isHiddenUser } = this.getProperties('isNeedFeedback', 'isHiddenUser');
+      if (isHiddenUser === null) {
```

Comment:
do we need to explicitely check for null (i.e. undefined should not pass? ) 
Other wise we can just do `if (isHiddenUser)`

Reply:
it's a tri-state variable - "default", "true", "false". null is the default value. spelling all of this out as an enum seems overkill